src/rust/devcontainer-feature.json

```
@@ -1,6 +1,6 @@
 {
     "id": "rust",
-    "version": "1.0.12",
+    "version": "1.0.13",
```

```suggestion
    "version": "1.1.0",
```

Can we bump the minor version instead?

We usually bump major versions for breaking changes, however, this doesn't seem like a breaking change to me. Hence, bumping minor version should be fine.